fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/insert/AbstractInsertExecutor.java

```diff
@@ -194,19 +199,14 @@ public void executeSingleInsert(StmtExecutor executor, long jobId) throws Except
             execImpl(executor, jobId);
             checkStrictModeAndFilterRatio();
             int retryTimes = 0;
-            while (true) {
+            while (retryTimes < Config.mow_insert_into_commit_retry_times) {
                 try {
                     onComplete();
                     break;
                 } catch (UserException e) {
-                    LOG.warn("failed to commit txn, txnId={}, jobId={}, retryTimes={}",
-                            getTxnId(), jobId, retryTimes, e);
+                    LOG.warn("failed to commit txn", e);
                     if (e.getErrorCode() == InternalErrorCode.DELETE_BITMAP_LOCK_ERR) {
```

Comment:
The logic you modified does not equal to before.
Before: after retry several times, it will throw an exception.
After: after retry several times, no exception will be thrown.